### DISCUSSION TITLE
Please help - I am so confused on how to design using the Tailwind CSS Figma file

### DISCUSSION THREAD

**Discussion by Mimi1111111** [tailwindlabs/tailwindcss#12516] [Help]
- Where can I find a full tailwind css figma file that has all of the components that are available: breadcrumbs, layout of the dashboard etc, as well as the colors and spacing. I keep finding small kits that have just the color, text hierarchy, and shadows.
- Could you please link the full figma file.
I also do not understand how I can go about designing using the Tailwind CSS library. Do I even need to do a Figma file?
- Can I just not drag and drop pieces of code and by using the opinionated one, I'd be able to place them together like a no code builder.

I am very confused. We already made components in Tailwind using custom components. But we are facing padding and spacing issues and want to understand if it is even a good idea to go to the opinionated Tailwind now. 

- If so how can we take our functionality and inject it to the Tailwind CSS pre-defined components?

**Comment by wongjn** [comment]
Are you perhaps talking about *Tailwind UI*? If so, [its Figma file was discontinued](https://tailwindui.com/documentation#figma-assets). *Tailwind CSS* does not include any UI components.

**Reply by Mimi1111111** [reply]
Yes am talking about the Tailwind UI. So now if I wanted to use the tailwind css component out of the box how would I go about doing that? Would I just drag and drop code snippets into place where I want it to?
Would I even need a ux or a ui designer for this given that they discontinued the ui components? I saw a bunch of devs just dragging whatever they wanted into the general place and followed a structure. It never shows them following a design file. An example of what I find: https://www.youtube.com/watch?v=qmMIz0_c9Ek

I am not a dev and am a product owner so am confused on how Tailwind can be used if there is no design system for it. Could you please clarify.

**Reply by wongjn** [reply]
Tailwind UI is a collection of webpage templates using Tailwind CSS. You can copy the majority of the code into a project, though some integration adjustments might need to be made. I would say it is not a no-code solution, but starting points for projects.
